elasticsearch/examples/upgrade/Makefile

```diff
@@ -3,15 +3,12 @@ default: test
 include ../../../helpers/examples.mk
 
 RELEASE := helm-es-upgrade
+KUBE_MINOR_VERSION := $(shell kubectl version -o json | jq -c '.serverVersion.minor' | sed 's/[^0-9]*//g')
```

Comment:
I know this is just an example, but should the requirement for `jq` be documented somewhere?

Reply:
This shell logic seems like a prime candidate to pull out into a real script due to not being able to do error handling here. If `jq` does not exist `7.4.0` will be used. These make files are only used in the testing framework but it could be worth calling out as a dependency.

Reply:
I replaced `jq` by `grep` to avoid adding some dependency.

@Conky5, agree with error handling, we can have a script like this to get Elasticsearch version:

```shell
KUBE_MINOR_VERSION=$(kubectl version --client -o yaml | grep minor | sed 's/[^0-9]*//g')

if [ "$KUBE_MINOR_VERSION" -lt 16 ]
then
  echo 7.0.0-alpha1
else
  echo 7.4.0
fi
```

then use `VERSION := $(shell sh ./find-version.sh)` into the Makefile.

WDYT?

Reply:
I would recommending pulling the entire `install` target out into a shell script like `./scripts/install.sh` and including these version computations there.